portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,59 @@
-#content {
+.content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  text-align: center;
 }
+nav {
+    list-style-type: none;
+    background-color: lightblue;
+    border: solid;
+    margin-bottom: 20px;
+}
+
+footer {
+    list-style-type: none;
+    background-color: lightblue;
+    border: solid;
+    height: 1%;
+}
+
+.project-image {
+    width: 500px;
+    height: 300px;
+}
+
+ul#footer {
+    list-style: none;
+}
+
+ul#footer li{
+    display: inline;
+    font-family: serif;
+    font-size: 15px;
+    margin: 50px
+}
+
+ul#navigation {
+    list-style:none;
+}
+ul#navigation li {
+    display: inline;
+    padding: 20px;
+    font-family: serif;
+    font-size: 20px;
+}
+
+.alt1{
+   background-color: pink;
+}
+
+.alt2{
+   background-color: beige;
+}
```

Comment:
tip: you could use even/odd styling for this

https://www.w3.org/Style/Examples/007/evenodd.en.html

e.g. just make all the divs of class "section" and in CSS:

```css
.section:nth-child(even) {
  background-color: pink;
}
.section:nth-child(odd) {
  background-color: beige;
}
```